### ISSUE TITLE
Questions/Comments/Feedback thread

### ISSUE THREAD

**Issue by anka-213** [anka-213/webcomic_reader#38]
A place for generic feedback and questions, anything goes. For bug-reports, site requests and feature requests, please open a new issue.

**Comment by Ampelioides** [comment]
I'm having some problems getting achewood to work. The little dialog to say I was new to using the script triggered, and I went through the settings (half-understanding most of them) but I can't get any of the features to work on achewood itself. 

As an aside, does the script have a way to display alt-texts on comics?

**Comment by anka-213** [comment]
It should display the alt-text automatically by default. I have updated the code for the site now, so it should work if you update the script.

But yes, the settings are a bit over-complicated. I should probably add a simplified version and hide the other options by default.

**Comment by Ampelioides** [comment]
Ah works perfectly now, cheers.

**Comment by Tmp341** [comment]
Can we make this multilanguage?

**Comment by anka-213** [comment]
@Tmp341 Make what multilanguage? Do you want to translate the script to some other language? I so, I can add some framework to simplify the process.

**Comment by Tmp341** [comment]
Script i mean. Like, we can use "russian word for next" other than "english next".

**Comment by anka-213** [comment]
@Tmp341 If you want to translate the script to your language, I can start working on adding a localization framework to simplifying the process. It will be some work, but it shouldn't be too bad.

It might force some restructuring though, since we'll probably want the translations in separate files so that changing the translations won't cause a conflict with changing the main script. I probably want to split the script to put the site implementations in a separate file as well, and then use a script to build the whole project together. In other words, this change might take some time.

---

On the other hand, if we want to go with the lazy solution (at least for me ;-) ), you can just fork the repository and just translate all (displayed) strings in the copy. That will of course cause more work in the long term, since you will have to merge all upstream changes to keep the translated copy updated. 

If so, we could create a shared repository so I can handle most of the merges. There shouldn't be too much  merge conflicts.

---

We might also go with a compromise between these paths. For example, you can start translating in a fork and then later, when we have the framework I can extract the translations and put them in a separate translation-file so we'll avoid the merge conflicts.

Feel free to ask if any of that was unclear or if you have other questions (or if I completely misinterpreted you).

**Comment by Tmp341** [comment]
Nope. No problem here. But here is a problem, i'll be in military service for 1 year, until November 2017. Before that, i can't do anything. I'll join my team next week. But you can keep the idea.
